packages/ods/src/components/pagination/src/components/ods-pagination/ods-pagination.scss

```diff
@@ -0,0 +1,82 @@
+:host(.ods-pagination) {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+}
+
+:host([isDisabled]) {
+  opacity: .5;
+  cursor: not-allowed;
+}
+
+.ods-pagination {
+  &__results,
+  &__list {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 0;
+    list-style-type: none;
+  }
+
+  &__results {
+    gap: 8px;
+    color: var(--ods-color-primary-500);
+
+    &--disabled {
+      color: var(--ods-color-neutral-500);
+    }
+  }
+
+  &__list {
+    &__page,
+    &__arrow {
+      display: flex;
+      align-items: center;
+    }
+
+    &__arrow {
+      padding: 0 8px;
+
+      &__button::part(button) {
+        display: flex;
+        justify-content: center;
+        width: 2.5rem;
```

Comment:
could be nice to use a sass var to share this value everywhere here `$ods-pagination-button-width`

Reply:
use `pagination.$ods-pagination-button-width;`